content/en/how-to-guides/configuration-management/change_management.adoc

```diff
@@ -0,0 +1,62 @@
+
+. Write or update the appropriate documentation page relevant to the design changes to be made in your working branch.  All documentation should be nested under link:../../tutorials[Tutorials], link:../../how-to-guides[How-To Guides], link:../../explanations[Explanations], and/or link:../../references[References]. (See <<#_how_to_write_good_documentation, How to write good documentation>>)
+
+. Commit your changes and create a pull request (PR). Prepend the pull request title with the keyword for closing an issue, followed by the issue number, then issue description, e.g., `Closes #1701: add tricorder with touchscreen docs`. If the pull request closes multiple issues, simply reference each issue prepended with a closing keyword. (See https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords[closing issues using keywords].
+
+. If you're not yet ready for a review, mark the PR as _Draft_ to indicate it's a work in progress. Continue updating your doc and pushing your changes until you're happy with the content.
+
+. When you're ready for a review, mark the PR as _Ready for review_.
+
+== How to make design changes ==
+
+Design changes include changes to CAD, STL, Drawings, or software.
+
+. Update the documentation to reflect the design changes to be made. (See <<_how_to_make_documentation_changes, How to make documentation changes>>)
+
+. Create a new working branch, or checkout an existing branch, in the https://github.com/tetrabiodistributed/project-tetra[Project Tetra repo]. The branch name should describe the change to be made, followed by its associated issue number, e.g., `add-tricoder-with-touchscreen-1031`. If no such issue exists, create a new issue in the https://github.com/tetrabiodistributed/project-tetra/issues[Project Tetra issues].
++
+Note: Part number revisions and version releases should accompany a design change where appropriate. (See link:../version_control/[Version Control])
+
+. Commit your changes and create a pull request (PR). Prepend the pull request title with the keyword for closing an issue, followed by the issue number, then issue description, e.g., `Closes #1031: add tricorder with touchscreen`. If the pull request closes multiple issues, simply reference each issue prepended with a closing keyword. (See https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords[closing issues using keywords].
+
+. If you're not yet ready for a review, mark the PR as _Draft_ to indicate it's a work in progress. Continue updating your doc and pushing your changes until you're happy with the content.
+
+. When you're ready for a review, mark the PR as _Ready for review_.
+
```

Comment:
In an attempt to acknowledge that the above PR workflow may not suit the traditional mechanical design workflow, I'm debating adding the following note, but it also seems a bit redundant. 
_Note: It is the pull request author and reviewer's shared responsibility to ensure that the changes being proposed in the pull request are accurate._

Reply:
I think it was brought up at today's meeting that we should have some kind of final approver to review everything. For now this is good, but we'll have to figure out something eventually

Reply:
I concur. Let's continue the discussion in issue #37